src/public/media.ts

```diff
@@ -0,0 +1,74 @@
+import { GlobalVars } from '../internal/globalVars';
+import { SdkError, ErrorCode } from './interfaces';
+import { ensureInitialized, sendMessageRequestToParent, isAPISupportedByPlatform } from '../internal/internalAPIs';
+import { frameContexts, captureImageMobileSupportVersion } from '../internal/constants';
+
+/**
+ * Enum for file formats supported
+ */
+export enum FileFormat {
+  Base64 = 'base64',
+}
+
+/**
+ * File object that can be used to represent image or video or audio
+ */
+export interface File {
+  /**
+   * Content of the file
+   * App needs to convert this to dataUrl, if this must be used directly in HTML tags
+   */
+  content?: string;
```

Comment:
Shouldn't these be mandatory fields in File ?

Reply:
Yes. Initially we had error with every file, at which point everything was non-mandatory. Now that we moved error out, few of these are mandatory. Thanks for calling this out.